### ISSUE TITLE
JSON directory listing indicates type=directory for empty file entries

### ISSUE THREAD

**Issue by carueda** [joseluisq/static-web-server#270]
**Describe the bug**

JSON listing entry indicates `"type": "directory"` for empty files

**To Reproduce**

```
$ mkdir sws-base-dir && cd $_
$ mkdir -p public/dir1
$ touch public/file1
$ echo "hola" > public/file2
$ static-web-server --directory-listing=true --directory-listing-format=json
```

In other terminal, request `http://localhost/`, for example, via curlie, one gets:

```json
[
    {
        "name": "file2",
        "type": "file",
        "mtime": "2023-10-11T20:07:05Z",
        "size": 5
    },
    {
        "name": "file1",
        "type": "directory",
        "mtime": "2023-10-11T20:06:50Z"
    },
    {
        "name": "dir1",
        "type": "directory",
        "mtime": "2023-10-11T20:06:38Z"
    }
]
```

**Expected behavior**

The entry indicates `"type": "file"` for empty file, eg the "file1" above.

**Environment and Specs**
 - static-web-server: v2.22.1

(btw, SWS is awesome!)

**Comment by joseluisq** [comment]
Good catch! Just fixed by #271.
Thanks for reporting!

**Comment by joseluisq** [comment]
Resolved in [v2.23.0](https://github.com/static-web-server/static-web-server/releases/tag/v2.23.0)
